lib/models/LifecycleConfiguration.js

```diff
@@ -672,9 +672,15 @@ class LifecycleConfiguration {
                     return Tag;
                 }).join('');
             }
-            const Filter = ((rulePrefix && tags) || (tags && tags.length > 1)) ?
-                `<Filter><And>${Prefix}${tagXML}</And></Filter>` :
-                `<Filter>${Prefix}${tagXML}</Filter>`;
+            let Filter;
+            if (rulePrefix && !tags) {
+                Filter = Prefix;
+            } else if ((rulePrefix && tags) || (tags && tags.length > 1)) {
```

Comment:
You may replace with the slightly simpler form `if (tags && (rulePrefix || tags.length > 1))`